apps/evm/lib/evm/configuration.ex

```diff
@@ -17,25 +17,25 @@ defmodule EVM.Configuration do
   # EIP7
   @callback has_delegate_call?(t) :: boolean()
 
-  # EIP150
+  # TangerineWhistle
   @callback extcodesize_cost(t) :: integer()
 
-  # EIP150
+  # TangerineWhistle
   @callback extcodecopy_cost(t) :: integer()
 
-  # EIP150
+  # TangerineWhistle
   @callback balance_cost(t) :: integer()
 
-  # EIP150
+  # TangerineWhistle
```

Comment:
What do you think about leaving EIP numbers in comments?

Reply:
Ah yep, I had just done a find and replace 👍 -> [Leave EIP numbers in comments](https://github.com/poanetwork/mana/pull/462/commits/5a1ab993ce2ba64a4f5ac9724b1157cbc232dd2a)